media/css/newsletter/newsletter-developer.scss

```diff
@@ -0,0 +1,231 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../pebbles/includes/lib';
+
+.content {
```

Comment:
Oh I see, this is my own fault because I did it first in `newsletter-mozilla` when pebbles was just forming. Still, I think we can eliminate the redundancy here and clean up the other page later :)